e2e/playwright_test/tests/createGame.spec.ts

```diff
@@ -79,23 +78,20 @@ test('Create a simple flow game successfully', async ({ page }) => {
   await channelAdminPage.clickSaveRowChannelIcon('White', 'Game Control');
   await channelAdminPage.clickSaveChannelButton();
   await channelAdminPage.verifySaveChannelSuccess();
-  await channelAdminPage.verifyChannelIsCreated("Red HQ");
 
   await channelAdminPage.clickaddAChannelBtn();
   await channelAdminPage.inputNameChannel("Blue Chat");
   await channelAdminPage.inputAParticipantChannelInfors('Blue', '', '');
   await channelAdminPage.clickSaveRowChannelIcon('Blue', '');
   await channelAdminPage.clickSaveChannelButton();
   await channelAdminPage.verifySaveChannelSuccess();
-  await channelAdminPage.verifyChannelIsCreated("Blue Chat");
```

Comment:
Hello @Tristina1788  - why have we stopped checking that these channels get created?

Reply:
Hi @IanMayo , the code should not remove for verifying the channels are created. Because I works on 2 machine so somehow I miss that code. I think maybe I forgot pull before continue working. Sorry for that.